README.md

```diff
@@ -399,6 +400,21 @@ See [Releases](https://github.com/node-saml/passport-saml/releases) to find the
 
 ## FAQ
 
+### How do i migrate from `passport-saml`
+
+The original `passport-saml` is now deprecated. To migrate to this version of `passport-saml` you'll need to use the following options which have a new default behaviour.
```

Comment:
Doesn't this advice contradict your note above?

At any rate, we don't want to suggest turning off all security because we don't want any copy-paste security issues for those that use this library. You can describe the problem, but I would advise againt code that can be copied.